package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "start": "npx tsc && cp -R views dist && cp -R public dist && cp -R images dist && node dist/app.js",
     "start-win": "npx tsc && copy -R views dist && copy -R public dist && copy -R images dist && node dist/app.js",
+    "dev": "ts-node-dev app.ts",
```

Comment:
Is this used anywhere?

Reply:
no i was just checking it out lol i'll remove it and its dependencies now